app/frontend/src/components/forms/submission/NotesPanel.vue

```diff
@@ -82,13 +83,19 @@ export default {
     type="list-item-two-line"
     class="bgtrans"
   >
-    <div class="mt-6 d-flex flex-md-row justify-space-between">
+    <div
+      class="d-flex flex-md-row justify-space-between"
+      @click="showNotesContent = !showNotesContent"
+    >
       <div cols="12" sm="6">
         <h2 class="note-heading" :lang="lang">
           {{ $t('trans.notesPanel.notes') }}
+          <v-icon>{{
+            showNotesContent ? 'mdi:mdi-chevron-down' : 'mdi:mdi-chevron-right'
```

Comment:
For LTR languages, would this be better as `mdi:mdi-chevron-left` when it's "closed"?

Reply:
Great suggestion, I didn't even think of that.